.eslintrc

```diff
@@ -43,6 +44,7 @@
     "@typescript-eslint/semi": [1, "never"],
 
     "no-console": 0,
+    "@typescript-eslint/strict-boolean-expressions": [1, {"allowString": false, "allowNumber": false, "allowNullableObject": false}],
```

Comment:
Can you please group this warning with the existing one `@typescript-eslint/semi`? All these others are actually disabled.

Reply:
> Can you please group this warning with the existing one `@typescript-eslint/semi`? All these others are actually disabled.

Sorry, I'm not following. Are you suggesting to change it to:

`"@typescript-eslint/strict-boolean-expressions": [1, "never"]`

> You can also make it be flagged as an error instead of a warning, in the same way we disallow `==`/`!=`.

👍

Reply:
That's nice, but now it's incorrectly grouped again 😅  Rules are currently grouped into level 2 (error), level 1 (warning) and level 0 (disable). You bumped this to an error, so you should put it near `eqeqeq` and `no-var`.

Also, let's be consistent on how to declare levels - we can use either numbers (0-2) or strings ("error", "warning"...), but let's not mix both.

Reply:
Ahhhh!!! Now I see it! 😅